src/Core/Configurations/RuntimeConfigProvider.cs

```diff
@@ -40,12 +40,12 @@ public class RuntimeConfigProvider
     /// </summary>
     public string? ManagedIdentityAccessToken { get; private set; }
 
-    private readonly RuntimeConfigLoader _runtimeConfigLoader;
+    private readonly FileSystemRuntimeConfigLoader _runtimeConfigLoader;
     private RuntimeConfig? _runtimeConfig;
 
     public string RuntimeConfigFileName => _runtimeConfigLoader.ConfigFileName;
 
-    public RuntimeConfigProvider(RuntimeConfigLoader runtimeConfigLoader)
+    public RuntimeConfigProvider(FileSystemRuntimeConfigLoader runtimeConfigLoader)
```

Comment:
Shouldn't that take a `RuntimeConfigLoader` now, so it's the base class?

Reply:
this is fixed. missed this earlier with some merge conflict fixes.